app/main.py

```diff
@@ -1 +1,57 @@
-#  write your code here
+from typing import List
+
+
+class Animal:
+    """
+    Class  that represents animal that can eat.
+    """
+    def __init__(self,
+                 name: str,
+                 appetite: int,
+                 is_hungry: bool = True
+                 ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self):
+        """Function that prints information about animal name."""
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self):
+        """Function for eating given number of point."""
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+
+        return 0
```

Comment:
The `else` statement is not necessary here. According to the checklist, you should avoid using an extra `else` statement when the `if` block contains a `return` statement. You can directly return `0` after the `if` block.